docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/02-create-key.png" alt="Creating a new AWS key pair"/>
+            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>flocker-test</strong>. You'll need this later.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/03-pem-downloaded.png" alt="A downloaded pem file"/>
+            <span>The private key (.pem file) will be downloaded onto your computer.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 2</h2>
+			<p>Create a Flocker cluster using our CloudFormation template:
+            <br />
+			<a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?templateURL=https:%2F%2Fs3.amazonaws.com%2Finstaller.downloads.clusterhq.com%2Fflocker-cluster.cloudformation.json" class="button" target="_blank" align="middle">Launch Flocker CloudFormation</a>
+            <br />
+			This button will open CloudFormation in a new window.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/11-cloudformation-stackname.png" alt="Specifying the stack name"/>
+			<span>Click "Next". Enter a <code>Stack name</code>. This can be any descriptive name.</span>
+			<span>Enter your <code>KeyName</code> from Step 1. Then enter your AWS <code>AccessKeyID</code> and <code>SecretAccessKey</code> credentials.</span><span>If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.</span>
+            <div id="iam-instructions" style="text-align:left; display:none;">
+                <span>You can generate new credentials on your <a href="https://console.aws.amazon.com/iam/home#users" target="_blank">IAM Users</a> page:</span>
+                <span><ul><li>Click on your user and go to the "Security Credentials" tab.</li><li>Click "Create Access Key".</li><li>Click "Show User Security Credentials".</li></ul></span>
+            </div>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/12-cloudformation-settings.png" alt="Optionally fill in volumehub settings"/>
+            <span>Optionally, <a href="https://clusterhq.com/volumehub/" target="_blank">register for a Volume Hub account</a>. This provides a web interface to see what's going on in your cluster.</span>
+           <span>Once you're logged in to the Volume Hub, <a href="https://volumehub.clusterhq.com/v1/token" target="_blank">get your Volume Hub token here</a> and copy the <code>&lt;YourToken&gt;</code> part from <code>{"token": "&lt;YourToken&gt;"}</code> into the <code>VolumeHubToken</code> field.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/13-cloudformation-create.png" alt="Click create"/>
+			<span>Click "Next" twice and then click "Create" to create your cluster.</span>
```

Comment:
`Click "Next" twice and then click "Create"` 
> 
```
Complete the remaining CloudFormation configuration pages;  
use the default settings unless you are familiar with CloudFormation. 
Finally click "Create" to create your cluster.

.. note:: it will take 5-10 minutes for the cluster to boot.
```